config/locales/views/users/he.yml

```
@@ -0,0 +1,97 @@
+      heading: "התראות"
+      all: "הכל"
+      unread: "לא נקרא"
+      mark_as_read_button: "סמן כנקרא"
+      no_notifications: "אין לך התראות!"
+      no_unread_notifications: "אין לך התראות שלא נקראו!"
+      see_all: "ראה הכל"
+      fork_notification: "%{user} ביצע פיצול לפרויקט שלך %{project}"
+      star_notification: "%{user} הוסיף כוכב לפרויקט שלך %{project}"
+      new_assignment_notification: "הוקצה לך מטלה חדשה '%{assignment_name}'"
+      forum_thread_notification: "%{user} פרסם אשכול חדש '%{thread}'"
+      comment_notif: "%{user} הגיב באשכול '%{thread}'"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)